modules/apps/remote-app/remote-app-service/src/main/java/com/liferay/remote/app/internal/search/RemoteAppEntryIndexer.java

```diff
@@ -103,7 +105,17 @@ protected Document doGetDocument(RemoteAppEntry remoteAppEntry)
 				name);
 		}
 
-		document.addText(Field.URL, remoteAppEntry.getUrl());
+		String type = remoteAppEntry.getType();
+
+		if (type.equals(RemoteAppConstants.TYPE_CUSTOM_ELEMENT)) {
+			String customElementURLS = remoteAppEntry.getCustomElementURLs();
+
+			document.addText(
```

Comment:
Do we need to index this? or this is just for completion?

Reply:
It's for the dataset view. However we (@jbalsas and me) were considering removing the URL field from the dataset in the future, since it is a field which makes some sense for remote apps but not for custom elements.

In any case, using good names for both (custom elements and remote apps) should make it redundant, so I would say we may remove it in the end...